### PR TITLE
Add missing docs for timeout parameter in WebSocketResponse.__init__()

### PR DESCRIPTION
**Pull request by lenard-mosys**
## What do these changes do?

Adds missing documentation.

## Are there changes in behavior for the user?

No.

## Is it a substantial burden for the maintainers to support this?

No

## Related issue number

This was easier than opening an issue.

## Checklist

- [x] I think the code is well written
- [ ] ~~Unit tests for the changes exist~~
- [x] Documentation reflects the changes
- [ ] ~~If you provide code modification, please add yourself to `CONTRIBUTORS.txt`~~
- [ ] Add a new news fragment into the `CHANGES/` folder

### CODE REVIEW

File: docs/web_reference.rst [hunk 1]
```diff
@@ -953,12 +953,18 @@ and :ref:`aiohttp-web-signals` handlers::
...
 
+   :param float timeout: Timeout value for the `close`
+                         operation. After sending the close websocket message,
+                         `close` waits for ``timeout`` seconds for a response.
+                         Default value is ``10.0`` (10 seconds for `close`
+                         operation)
+
    :param float receive_timeout: Timeout value for `receive`
-                                 operations.  Default value is None
+                                 operations.  Default value is ``None``
```
Comment (webknjaz) [inline docs/web_reference.rst]:
```suggestion
                                 operations.  Default value is :data:`None`
```

File: docs/web_reference.rst [hunk 2]
```diff
@@ -953,12 +953,18 @@ and :ref:`aiohttp-web-signals` handlers::
                            connection if `pong` response is not
                            received. The timer is reset on any data reception.
 
+   :param float timeout: Timeout value for the `close`
+                         operation. After sending the close websocket message,
+                         `close` waits for ``timeout`` seconds for a response.
+                         Default value is ``10.0`` (10 seconds for `close`
```
Comment (webknjaz) [inline docs/web_reference.rst]:
```suggestion
                         Default value is ``10.0`` (10 seconds for ``close``
```

File: docs/web_reference.rst [hunk 3]
```diff
@@ -953,12 +953,18 @@ and :ref:`aiohttp-web-signals` handlers::
                            connection if `pong` response is not
                            received. The timer is reset on any data reception.
 
+   :param float timeout: Timeout value for the `close`
+                         operation. After sending the close websocket message,
+                         `close` waits for ``timeout`` seconds for a response.
```
Comment (webknjaz) [inline docs/web_reference.rst]:
```suggestion
                         ``close`` waits for ``timeout`` seconds for a response.
```

File: docs/web_reference.rst [hunk 4]
```diff
@@ -953,12 +953,18 @@ and :ref:`aiohttp-web-signals` handlers::
                            connection if `pong` response is not
                            received. The timer is reset on any data reception.
 
+   :param float timeout: Timeout value for the `close`
```
Comment (webknjaz) [inline docs/web_reference.rst]:
```suggestion
   :param float timeout: Timeout value for the ``close``
```

File: docs/web_reference.rst [hunk 5]
```diff
@@ -953,12 +953,18 @@ and :ref:`aiohttp-web-signals` handlers::
...
+                         operation)
+
    :param float receive_timeout: Timeout value for `receive`
-                                 operations.  Default value is None
+                                 operations.  Default value is ``None``
                                  (no timeout for receive operation)
 
    :param bool compress: Enable per-message deflate extension support.
-                          False for disabled, default value is True.
+                          False for disabled, default value is ``True``.
```
Comment (webknjaz) [inline docs/web_reference.rst]:
```suggestion
                          :data:`False` for disabled, default value is :data:`True`.
```

**Review by bdraco — APPROVED** [review]

**Comment by bdraco** [comment]
Thanks @lenard-mosys

**Pull request merged**